packages/@okta/vuepress-theme-prose/global-components/EventTypes.vue

```diff
@@ -81,9 +81,15 @@
           :key="tag"
           class="event-type-tag"
         >{{ tag }}</code>
+        <code
+          v-if="eventType.schema.changeDetails !== null && (!Array.isArray(eventType.schema.changeDetails) || eventType.schema.changeDetails.length > 0) "
```

Comment:
By looking at event-types.json, changeDetails can be an array or null. We want to show new tag when array is not empty. Although your current check works, `!Array.isArray(eventType.schema.changeDetails)` is always false and doesn't add much to verification.
It probably should be when details is not null also details is an array also the array is not empty:
```
v-if="eventType.schema.changeDetails !== null && Array.isArray(eventType.schema.changeDetails) && eventType.schema.changeDetails.length > 0) "
```

Reply:
Thanks, appreciated the feedback. Implemented your update.